unlock-protocol.com/blog/version-1-2.md

```diff
@@ -0,0 +1,87 @@
+
+By using Etherscan you can also deploy locks, without having to go through [our Dashboard](https://app.unlock-protocol.com/dashboard/), using the `createLock` method directly. The new version deployed on Monday uses the new Ethereum `create2` and leverages [eip-1014](https://eips.ethereum.org/EIPS/eip-1014) under the hood. This allows creators to deploy locks at only 15% of the cost that it took before that! In order to achieve this, we had to deploy a so [called `template` contract](https://etherscan.io/address/0x2d5487420fbeb5ba74eadf51084d4f71e1733983) which provides the reference implementation for locks to leverage.
+
+
+# The new public locks
+
+I want to repeat that old the **locks deployed before Monday have stayed absolutely unchanged**. It is actually impossible to change them. Once deployed, their code cannot be altered. This is a very important part of the "decentralization promise" that we make to creators: nobody will be able to change the terms of your memberships, but them.
+
+Now, if someone deploys a lock now, it comes with new features and improvements. Here are a few of the most exciting ones.
+
+## On Key Sold hook
+
+We strongly believe that one of the most important benefits of building on Ethereum is the **composability**. In practice, this is the ability for contracts to interact for the benefits of users. For example, purchasing a key using an ERC20 is an example of composability. Now, with hooks, the lock owner can define what other contracts will be able to run when a key has been sold.
+
+Why is this useful? Here are 2 of my favorite examples:
+
+1. _Matching donations_: someone could agree to match donations so that for any key purchase on a donation lock, they would make a payment to the lock as well. The "matching" contract could even query the lock itself to know how many keys have been sold and make a quadratic matching!
+
+2. _A bonded curve lock_: a hook could increase the price of keys on a lock based on how many keys have been sold in the past. This can create bonding curves where early members end up paying less than late members, while also having the ability to re-sell their early membership at a higher price. You can easily use a lock as a [Token Curated Registry](https://medium.com/@ilovebagels/token-curated-registries-1-0-61a232f8dac7) where only members of the lock can change the data of the registry!
+
+In practice, there is an endless amount of possibilities here and we cannot wait to see what people are building.
```

Comment:
Could be trimmed for a better flow:
"In practice, there are endless possibilities here and we cannot wait to see what people are building."